<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="4" ignored="0" total="4" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="MiLuzTestSuite" duration-ms="13659" started-at="2018-11-22T09:39:36Z" finished-at="2018-11-22T09:39:50Z">
    <groups>
    </groups>
    <test name="MantenimientoProviderChrome" duration-ms="13659" started-at="2018-11-22T09:39:36Z" finished-at="2018-11-22T09:39:50Z">
      <class name="pe.com.test.system.testCase.MantenimientoProvider">
        <test-method status="PASS" signature="antesDelTest(java.lang.String, int, int, java.lang.String)[pri:0, instance:pe.com.test.system.testCase.MantenimientoProvider@1794d431]" name="antesDelTest" is-config="true" duration-ms="2805" started-at="2018-11-22T09:39:36Z" finished-at="2018-11-22T09:39:39Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[0]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[1]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[Chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- antesDelTest -->
        <test-method status="FAIL" signature="a_insertarProveedor(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:pe.com.test.system.testCase.MantenimientoProvider@1794d431]" name="a_insertarProveedor" duration-ms="9483" started-at="2018-11-22T09:39:40Z" data-provider="DatosDeEntradaInsertar" finished-at="2018-11-22T09:39:49Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Registrar Proveedor - Flujo Básico]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[http://localhost:8080/login]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[marcelo]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[123456]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[Samsung]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[987654321]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[Proveedor creado con éxito!]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[http://localhost:81/testlink/lib/api/xmlrpc/v1/xmlrpc.php]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[c2ba2f17a374eb08746d1a30bd837bf7]]>
              </value>
            </param>
            <param index="9">
              <value>
                <![CDATA[14]]>
              </value>
            </param>
            <param index="10">
              <value>
                <![CDATA[1]]>
              </value>
            </param>
            <param index="11">
              <value>
                <![CDATA[17]]>
              </value>
            </param>
            <param index="12">
              <value>
                <![CDATA[1]]>
              </value>
            </param>
            <param index="13">
              <value>
                <![CDATA[Build 01]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Error: null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Error: null
at org.testng.Assert.fail(Assert.java:96)
at pe.com.test.system.testCase.MantenimientoProvider.a_insertarProveedor(MantenimientoProvider.java:101)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- a_insertarProveedor -->
        <test-method status="FAIL" signature="a_insertarProveedor(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:pe.com.test.system.testCase.MantenimientoProvider@1794d431]" name="a_insertarProveedor" duration-ms="35" started-at="2018-11-22T09:39:49Z" data-provider="DatosDeEntradaInsertar" finished-at="2018-11-22T09:39:49Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Registrar Proveedor - Flujo Alternativo]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[http://localhost:8080/login]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[marcelo]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[123456]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[987654321]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[El nombre de proveedor no puede ser vacío]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[http://localhost:81/testlink/lib/api/xmlrpc/v1/xmlrpc.php]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[c2ba2f17a374eb08746d1a30bd837bf7]]>
              </value>
            </param>
            <param index="9">
              <value>
                <![CDATA[14]]>
              </value>
            </param>
            <param index="10">
              <value>
                <![CDATA[1]]>
              </value>
            </param>
            <param index="11">
              <value>
                <![CDATA[17]]>
              </value>
            </param>
            <param index="12">
              <value>
                <![CDATA[1]]>
              </value>
            </param>
            <param index="13">
              <value>
                <![CDATA[Build 01]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Error: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=70.0.3538.102)
  (Driver info: chromedriver=2.43.600210 (68dcf5eebde37173d4027fa8635e332711d2874a),platform=Windows NT 10.0.17134 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 0 milliseconds
Build info: version: '3.14.0', revision: 'aacccce0', time: '2018-08-02T20:19:58.91Z'
System info: host: 'LAPTOP-4QGIS7L0', ip: '192.168.48.1', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_191'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, acceptSslCerts: false, applicationCacheEnabled: false, browserConnectionEnabled: false, browserName: chrome, chrome: {chromedriverVersion: 2.43.600210 (68dcf5eebde371..., userDataDir: C:\Users\alvar\AppData\Loca...}, cssSelectorsEnabled: true, databaseEnabled: false, goog:chromeOptions: {debuggerAddress: localhost:58606}, handlesAlerts: true, hasTouchScreen: false, javascriptEnabled: true, locationContextEnabled: true, mobileEmulationEnabled: false, nativeEvents: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: XP, platformName: XP, rotatable: false, setWindowRect: true, takesHeapSnapshot: true, takesScreenshot: true, unexpectedAlertBehaviour: , unhandledPromptBehavior: , version: 70.0.3538.102, webStorageEnabled: true}
Session ID: d8b1266f7f8450740480daf56c1d8994]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Error: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=70.0.3538.102)
  (Driver info: chromedriver=2.43.600210 (68dcf5eebde37173d4027fa8635e332711d2874a),platform=Windows NT 10.0.17134 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 0 milliseconds
Build info: version: '3.14.0', revision: 'aacccce0', time: '2018-08-02T20:19:58.91Z'
System info: host: 'LAPTOP-4QGIS7L0', ip: '192.168.48.1', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_191'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, acceptSslCerts: false, applicationCacheEnabled: false, browserConnectionEnabled: false, browserName: chrome, chrome: {chromedriverVersion: 2.43.600210 (68dcf5eebde371..., userDataDir: C:\Users\alvar\AppData\Loca...}, cssSelectorsEnabled: true, databaseEnabled: false, goog:chromeOptions: {debuggerAddress: localhost:58606}, handlesAlerts: true, hasTouchScreen: false, javascriptEnabled: true, locationContextEnabled: true, mobileEmulationEnabled: false, nativeEvents: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: XP, platformName: XP, rotatable: false, setWindowRect: true, takesHeapSnapshot: true, takesScreenshot: true, unexpectedAlertBehaviour: , unhandledPromptBehavior: , version: 70.0.3538.102, webStorageEnabled: true}
Session ID: d8b1266f7f8450740480daf56c1d8994
at org.testng.Assert.fail(Assert.java:96)
at pe.com.test.system.testCase.MantenimientoProvider.a_insertarProveedor(MantenimientoProvider.java:101)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- a_insertarProveedor -->
        <test-method status="FAIL" signature="b_editarProveedor(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:pe.com.test.system.testCase.MantenimientoProvider@1794d431]" name="b_editarProveedor" duration-ms="14" started-at="2018-11-22T09:39:49Z" data-provider="DatosDeEntradaEditar" finished-at="2018-11-22T09:39:49Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Actualizar Proveedor - Flujo Básico]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[http://localhost:8080/MiLuz-0.0.1-SNAPSHOT/login]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[marcelo]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[123456]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[CocaCola]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[Proveedor editado con éxito!]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[http://localhost:81/testlink/lib/api/xmlrpc/v1/xmlrpc.php]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[c2ba2f17a374eb08746d1a30bd837bf7]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[14]]>
              </value>
            </param>
            <param index="9">
              <value>
                <![CDATA[1]]>
              </value>
            </param>
            <param index="10">
              <value>
                <![CDATA[17]]>
              </value>
            </param>
            <param index="11">
              <value>
                <![CDATA[1]]>
              </value>
            </param>
            <param index="12">
              <value>
                <![CDATA[Build 01]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Error: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=70.0.3538.102)
  (Driver info: chromedriver=2.43.600210 (68dcf5eebde37173d4027fa8635e332711d2874a),platform=Windows NT 10.0.17134 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 0 milliseconds
Build info: version: '3.14.0', revision: 'aacccce0', time: '2018-08-02T20:19:58.91Z'
System info: host: 'LAPTOP-4QGIS7L0', ip: '192.168.48.1', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_191'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, acceptSslCerts: false, applicationCacheEnabled: false, browserConnectionEnabled: false, browserName: chrome, chrome: {chromedriverVersion: 2.43.600210 (68dcf5eebde371..., userDataDir: C:\Users\alvar\AppData\Loca...}, cssSelectorsEnabled: true, databaseEnabled: false, goog:chromeOptions: {debuggerAddress: localhost:58606}, handlesAlerts: true, hasTouchScreen: false, javascriptEnabled: true, locationContextEnabled: true, mobileEmulationEnabled: false, nativeEvents: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: XP, platformName: XP, rotatable: false, setWindowRect: true, takesHeapSnapshot: true, takesScreenshot: true, unexpectedAlertBehaviour: , unhandledPromptBehavior: , version: 70.0.3538.102, webStorageEnabled: true}
Session ID: d8b1266f7f8450740480daf56c1d8994]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Error: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=70.0.3538.102)
  (Driver info: chromedriver=2.43.600210 (68dcf5eebde37173d4027fa8635e332711d2874a),platform=Windows NT 10.0.17134 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 0 milliseconds
Build info: version: '3.14.0', revision: 'aacccce0', time: '2018-08-02T20:19:58.91Z'
System info: host: 'LAPTOP-4QGIS7L0', ip: '192.168.48.1', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_191'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, acceptSslCerts: false, applicationCacheEnabled: false, browserConnectionEnabled: false, browserName: chrome, chrome: {chromedriverVersion: 2.43.600210 (68dcf5eebde371..., userDataDir: C:\Users\alvar\AppData\Loca...}, cssSelectorsEnabled: true, databaseEnabled: false, goog:chromeOptions: {debuggerAddress: localhost:58606}, handlesAlerts: true, hasTouchScreen: false, javascriptEnabled: true, locationContextEnabled: true, mobileEmulationEnabled: false, nativeEvents: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: XP, platformName: XP, rotatable: false, setWindowRect: true, takesHeapSnapshot: true, takesScreenshot: true, unexpectedAlertBehaviour: , unhandledPromptBehavior: , version: 70.0.3538.102, webStorageEnabled: true}
Session ID: d8b1266f7f8450740480daf56c1d8994
at org.testng.Assert.fail(Assert.java:96)
at pe.com.test.system.testCase.MantenimientoProvider.b_editarProveedor(MantenimientoProvider.java:148)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- b_editarProveedor -->
        <test-method status="FAIL" signature="c_eliminarProveedor(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:pe.com.test.system.testCase.MantenimientoProvider@1794d431]" name="c_eliminarProveedor" duration-ms="8" started-at="2018-11-22T09:39:49Z" data-provider="DatosDeEntradaEliminar" finished-at="2018-11-22T09:39:49Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Eliminar Proveedor - Flujo Básico]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[http://localhost:8080/login]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[marcelo]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[123456]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[Samsung]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[Proveedor eliminado con éxito]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[http://localhost:81/testlink/lib/api/xmlrpc/v1/xmlrpc.php]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[c2ba2f17a374eb08746d1a30bd837bf7]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[14]]>
              </value>
            </param>
            <param index="9">
              <value>
                <![CDATA[1]]>
              </value>
            </param>
            <param index="10">
              <value>
                <![CDATA[17]]>
              </value>
            </param>
            <param index="11">
              <value>
                <![CDATA[1]]>
              </value>
            </param>
            <param index="12">
              <value>
                <![CDATA[Build 01]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Error: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=70.0.3538.102)
  (Driver info: chromedriver=2.43.600210 (68dcf5eebde37173d4027fa8635e332711d2874a),platform=Windows NT 10.0.17134 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 0 milliseconds
Build info: version: '3.14.0', revision: 'aacccce0', time: '2018-08-02T20:19:58.91Z'
System info: host: 'LAPTOP-4QGIS7L0', ip: '192.168.48.1', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_191'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, acceptSslCerts: false, applicationCacheEnabled: false, browserConnectionEnabled: false, browserName: chrome, chrome: {chromedriverVersion: 2.43.600210 (68dcf5eebde371..., userDataDir: C:\Users\alvar\AppData\Loca...}, cssSelectorsEnabled: true, databaseEnabled: false, goog:chromeOptions: {debuggerAddress: localhost:58606}, handlesAlerts: true, hasTouchScreen: false, javascriptEnabled: true, locationContextEnabled: true, mobileEmulationEnabled: false, nativeEvents: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: XP, platformName: XP, rotatable: false, setWindowRect: true, takesHeapSnapshot: true, takesScreenshot: true, unexpectedAlertBehaviour: , unhandledPromptBehavior: , version: 70.0.3538.102, webStorageEnabled: true}
Session ID: d8b1266f7f8450740480daf56c1d8994]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Error: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=70.0.3538.102)
  (Driver info: chromedriver=2.43.600210 (68dcf5eebde37173d4027fa8635e332711d2874a),platform=Windows NT 10.0.17134 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 0 milliseconds
Build info: version: '3.14.0', revision: 'aacccce0', time: '2018-08-02T20:19:58.91Z'
System info: host: 'LAPTOP-4QGIS7L0', ip: '192.168.48.1', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_191'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, acceptSslCerts: false, applicationCacheEnabled: false, browserConnectionEnabled: false, browserName: chrome, chrome: {chromedriverVersion: 2.43.600210 (68dcf5eebde371..., userDataDir: C:\Users\alvar\AppData\Loca...}, cssSelectorsEnabled: true, databaseEnabled: false, goog:chromeOptions: {debuggerAddress: localhost:58606}, handlesAlerts: true, hasTouchScreen: false, javascriptEnabled: true, locationContextEnabled: true, mobileEmulationEnabled: false, nativeEvents: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: XP, platformName: XP, rotatable: false, setWindowRect: true, takesHeapSnapshot: true, takesScreenshot: true, unexpectedAlertBehaviour: , unhandledPromptBehavior: , version: 70.0.3538.102, webStorageEnabled: true}
Session ID: d8b1266f7f8450740480daf56c1d8994
at org.testng.Assert.fail(Assert.java:96)
at pe.com.test.system.testCase.MantenimientoProvider.c_eliminarProveedor(MantenimientoProvider.java:193)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- c_eliminarProveedor -->
        <test-method status="PASS" signature="despuesDelTest()[pri:0, instance:pe.com.test.system.testCase.MantenimientoProvider@1794d431]" name="despuesDelTest" is-config="true" duration-ms="658" started-at="2018-11-22T09:39:49Z" finished-at="2018-11-22T09:39:50Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- despuesDelTest -->
      </class> <!-- pe.com.test.system.testCase.MantenimientoProvider -->
    </test> <!-- MantenimientoProviderChrome -->
  </suite> <!-- MiLuzTestSuite -->
</testng-results>
